aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -362,4 +368,4 @@ DEFINE_DISPATCH(tanh_stub);
 DEFINE_DISPATCH(trunc_stub);
 DEFINE_DISPATCH(lgamma_stub);
 }
-} // namespace at
+} // namespace at
```

Comment:
Newlines needed I think.

Reply:
What do you mean by "Newlines needed"?

Reply:
> Newlines needed I think.

What do you mean by "Newlines needed"?

Reply:
See the PR diff, there are some symbols at the end of some files.

Reply:
Indeed—to elaborate, @yuyihe, for historical Unixy reasons, it is generally considered good etiquette for source code files to always end in a newline.